README.md

```diff
@@ -162,3 +162,23 @@ You can run automated tests, code analysis and Django configuration checks with:
 ```
 
 Always run this command before submitting code in a pull request and make sure that there are no warnings or failed tests.
+
+
+### Running MiddleMan
+
+You can run MiddleMan service with default values with:
+
+``` bash
+concent_api/manage.py middleman
+```
+
+You may add `-h` to check how to define non-default values.
+
+
+### Running Celery workers in development
+
+You can run Celery workers for Concent with:
+
+``` bash
+concent_api/celery -A concent_api worker -l info -Q concent,conductor,verifier
```

Comment:
Please use long option names in your examples. They're easier to remember and it's easier to what the values mean.

Reply:
:+1: done